src/modules/ekf2/EKF2.cpp

```diff
@@ -1840,16 +1796,24 @@ void EKF2::UpdateGyroCalibration(const hrt_abstime &timestamp)
 		if (_gyro_cal.last_us != 0) {
 			_gyro_cal.total_time_us += timestamp - _gyro_cal.last_us;
 
-			// consider bias estimates stable when we have accumulated sufficient time
+			// Start checking gyro bias estimates when we have accumulated sufficient calibration time
 			if (_gyro_cal.total_time_us > 30_s) {
+				if (!_gyro_cal.cal_available) {
+					PX4_DEBUG("%d gyro bias now stable", _instance);
```

Comment:
Accidentally pulled back in during rebase?

Reply:
ouch